static/src/questionnaires/QuestionnaireBodyCreate.vue

```diff
@@ -230,34 +231,44 @@ export default Vue.extend({
         console.error('Cannot moveQuestionUp from index', qIndex)
         return
       }
-      this.moveArrayElement(this.themes[themeIndex].questions, qIndex, qIndex - 1)
-      this.updateOrderFields(this.themes[themeIndex].questions)
-      const isMoveUp = true
-      this.animateQuestionSwap(
-        $('#theme-' + themeIndex + '-question-' + qIndex),
-        $('#theme-' + themeIndex + '-question-' + (qIndex - 1)),
-        isMoveUp,
-      )
+      this._swapQuestions(themeIndex, qIndex, qIndex - 1)
     },
     moveQuestionDown(themeIndex, qIndex) {
       console.debug('moveQuestionDown, theme', themeIndex, '- question ', qIndex)
       if (qIndex >= (this.themes[themeIndex].questions.length - 1)) {
         console.error('Cannot moveQuestionDown from index', qIndex)
         return
       }
-      this.moveArrayElement(this.themes[themeIndex].questions, qIndex, qIndex + 1)
+      this._swapQuestions(themeIndex, qIndex, qIndex + 1)
+    },
+    _swapQuestions(themeIndex, qIndexFrom, qIndexTo) {
```

Comment:
Minor comment.

Naming: What's our practice around names starting with underscore?

Reply:
Good question.
What I use it for : indicate the function is "private" to the component. When you use this component, you should not have to call the private function, it's only for internal purposes.

There is a style guide for Vue (I didn't know!) which recommends hiding functions like this : 
```
var myComp = {
  // ...
  methods: {
    publicMethod() {
      // ...
      myPrivateFunction()
    }
  }
}

function myPrivateFunction() {
  // ...
}
```
or, not as good but still ok, prefixing with $_compname (which seems like a huge annoying prefix to me):
```
var myGreatMixin = {
  // ...
  methods: {
    $_myGreatMixin_update: function () {
      // ...
    }
  }
}
```
https://vuejs.org/v2/style-guide/#Private-property-names-essential

Reply:
For this one, I used $_